src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends AbstractFigure {
+    private String color;
```

Comment:
The `color` field is redundant here because it is already defined in the `AbstractFigure` class. You should remove this field from the `Circle` class to avoid confusion and potential errors.